<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameWorkspace</class>
 <widget class="QMainWindow" name="FrameWorkspace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[ScDrv]</string>
  </property>
  <widget class="FrameFile" name="viewMoleculeFile_"/>
  <widget class="QMenuBar" name="barMenu_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_">
     <property name="title">
      <string>R&amp;ecent</string>
     </property>
     <addaction name="actionForget_"/>
     <addaction name="actionForgetAll_"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew_"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_"/>
    <addaction name="actionReload_"/>
    <addaction name="menuRecent_"/>
    <addaction name="separator"/>
    <addaction name="actionSave_"/>
    <addaction name="actionSaveAs_"/>
    <addaction name="separator"/>
    <addaction name="actionExit_"/>
   </widget>
   <widget class="QMenu" name="menuHelp_">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAbout_"/>
    <addaction name="actionAboutQt_"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo_"/>
    <addaction name="actionRedo_"/>
    <addaction name="separator"/>
    <addaction name="actionCut_"/>
    <addaction name="actionCopy_"/>
    <addaction name="actionPaste_"/>
    <addaction name="separator"/>
    <addaction name="actionClear_"/>
    <addaction name="actionClearAll_"/>
   </widget>
   <widget class="QMenu" name="menuView_">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuMolecule_">
     <property name="title">
      <string>&amp;Molecule</string>
     </property>
     <addaction name="actionMolFast_"/>
     <addaction name="separator"/>
     <addaction name="actionMolStick_"/>
     <addaction name="actionMolBalls_"/>
     <addaction name="actionMolSpace_"/>
     <addaction name="separator"/>
     <addaction name="actionMolBonds_"/>
     <addaction name="actionLabelAtoms_"/>
     <addaction name="actionRenderLabelsBond_"/>
    </widget>
    <addaction name="actionFullScreen_"/>
    <addaction name="separator"/>
    <addaction name="actionToggleLayout_"/>
    <addaction name="separator"/>
    <addaction name="menuMolecule_"/>
   </widget>
   <widget class="QMenu" name="menuTools_">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSourceEdit_"/>
    <addaction name="actionSourceCast_"/>
    <addaction name="separator"/>
    <addaction name="actionOptions_"/>
   </widget>
   <widget class="QMenu" name="menuWindows_">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <addaction name="menuFile_"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView_"/>
   <addaction name="menuTools_"/>
   <addaction name="menuWindows_"/>
   <addaction name="menuHelp_"/>
  </widget>
  <widget class="QStatusBar" name="barStatus_"/>
  <widget class="QToolBar" name="barTools_">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;New..</string>
   </property>
  </action>
  <action name="actionOpen_">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open..</string>
   </property>
  </action>
  <action name="actionPrint_">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp_">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionSave_">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit_">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout_">
   <property name="text">
    <string>&amp;About..</string>
   </property>
  </action>
  <action name="actionAboutQt_">
   <property name="text">
    <string>About &amp;Qt..</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionUndo_">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo_">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCut_">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy_">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste_">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionClear_">
   <property name="text">
    <string>C&amp;lear</string>
   </property>
  </action>
  <action name="actionClearAll_">
   <property name="text">
    <string>Cle&amp;ar all</string>
   </property>
  </action>
  <action name="actionReload_">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionForget_">
   <property name="text">
    <string>&amp;Forget</string>
   </property>
  </action>
  <action name="actionForgetAll_">
   <property name="text">
    <string>Forget all &amp;recent</string>
   </property>
  </action>
  <action name="actionFullScreen_">
   <property name="text">
    <string>&amp;Full screen</string>
   </property>
  </action>
  <action name="actionOptions_">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actionSaveAs_">
   <property name="text">
    <string>S&amp;ave as..</string>
   </property>
  </action>
  <action name="actionToggleLayout_">
   <property name="text">
    <string>Togg&amp;le layout</string>
   </property>
  </action>
  <action name="actionMolFast_">
   <property name="text">
    <string>Render &amp;fast</string>
   </property>
  </action>
  <action name="actionMolStick_">
   <property name="text">
    <string>&amp;Sticks</string>
   </property>
  </action>
  <action name="actionMolSpace_">
   <property name="text">
    <string>S&amp;pace fill</string>
   </property>
  </action>
  <action name="actionMolBalls_">
   <property name="text">
    <string>&amp;Ball-stick</string>
   </property>
  </action>
  <action name="actionMolBonds_">
   <property name="text">
    <string>Toggle bonds</string>
   </property>
   <property name="statusTip">
    <string>Toggle the bond display in the molecule</string>
   </property>
  </action>
  <action name="actionLabelAtoms_">
   <property name="text">
    <string>Atom labels</string>
   </property>
  </action>
  <action name="actionRenderLabelsBond_">
   <property name="text">
    <string>Bond labels</string>
   </property>
  </action>
  <action name="actionSourceEdit_">
   <property name="text">
    <string>&amp;Edit source..</string>
   </property>
  </action>
  <action name="actionSourceCast_">
   <property name="text">
    <string>&amp;Interpret..</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameFile</class>
   <extends>QTabWidget</extends>
   <header>FrameFile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
